scripts/log-capture

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+date=$(date +%F_%R)
+OUTDIR=/tmp/LogCapture-${date}
+ARCHIVE=${1:/tmp/LogCapture.tar.bz2}
```

Comment:
Just a small nitpick - I think this should also be called "log-capture" - for consistency.